assets/src/application/constants/momentFormats.ts

```diff
@@ -0,0 +1,3 @@
+export const defaultDateFormat = 'yyyy-MM-dd';
+
+export const defaultTimeFormat = 'HH:mm:ss';
```

Comment:
They are not momentJS formats, they are `date-fns` equivalent formats for what AntD time picker passes to `onChange`

Reply:
ahh ok... i got confused by this comment you made previously:

> Those formats are specific to AntD datepicker because they use momentJS. It can be consumed directly from EspressoForm when you use it.

from: https://github.com/eventespresso/event-espresso-core/pull/2412/files#r381373500

so these formats are used by us with date-fns to convert things to the same format use by moment.js when using the AntD datepicker.

I'll adjust this accordingly